### PR TITLE
adding Energy Web Foundation implementation

### PR DESCRIPTION
**Pull request by artursudnik**

### CODE REVIEW

**Review by msporny — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Great @artursudnik ! Thank you so much for adding your implementation! :)

File: implementations/EWF.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "implementation": "Energy Web Foundation VC-API",
+  "issuers": [{
+    "id": "did:key:z6MksoRPRqnMWiivV4weRGGS9SiHXuqfJEYu95EiYtbvBxW6",
+    "endpoint": "https://vc-api-dev.energyweb.org/v1/vc-api/credentials/issue",
+    "tags": ["vc-api", "Ed25519Signature2020"]
+
+  }],
+  "verifiers": [{
+    "id": "",
+    "endpoint": "https://bridge.learncard.com/credentials/verify",
```
Comment (jrhender) [inline implementations/EWF.json:12]: @artursudnik I just noticed this. Is this value correct?

Comment (artursudnik) [reply implementations/EWF.json:12]: Oh, no! This is my mistake when copy-pasting. I am fixing this.

Comment (artursudnik) [reply implementations/EWF.json:12]: https://github.com/w3c-ccg/vc-api-test-suite-implementations/pull/53

**Comment by msporny** [comment]
@artursudnik and @jrhender -- we'd like to feature EWF's Issuer and Verifier in the VC Issuer Playground:

https://playground.chapi.io/issuer

In order to do that, could you please include the following JSON-LD Contexts in your issuer implementation so that you can issue VCs for all current demo credentials? The list of contexts are here:

https://docs.google.com/document/d/1L3GnHtvrH51MJ7W2ZAL-PIfzR2dxLyFoDrUAHZ1KBU4/edit

**Comment by msporny** [comment]
> However, it would, in general, be valuable for EWF to be able to include additional contexts in our VC-API implementation, so I will re-prioritize this on our backlog, but it might not be in the near-term.

Understood and thank you for adding it to your backlog. We would love to feature you in the CHAPI VC Issuer Playground.

What we are considering is a way for you to specify which contexts you support so that the playground will automatically disable issuing VCs that you don't support.

I'll also note that we know of at least two other organizations that are using DIDKit and have added the contexts above to their builds. If it would help, we can put you in contact with those organizations?

**Comment by jrhender** [comment]
> I'll also note that we know of at least two other organizations that are using DIDKit and have added the contexts above to their builds. If it would help, we can put you in contact with those organizations?

Yes please, that does sound helpful. If you would like to do this via email, my email is john.henderson@energyweb.org

**Pull request merged**